src/IndexRegistry.sol

```diff
@@ -147,30 +138,24 @@ contract IndexRegistry is IndexRegistryStorage {
     function _processOperatorRemoval(
         bytes32 operatorId, 
         uint8 quorumNumber, 
-        uint32 indexOfOperatorToRemove, 
-        bytes32 operatorIdToSwap
+        uint32 indexOfOperatorToRemove 
     ) internal {   
-        uint32 operatorIdToSwapIndex = _operatorIdToIndexHistory[operatorIdToSwap][quorumNumber][_operatorIdToIndexHistory[operatorIdToSwap][quorumNumber].length - 1].index;
-        require(
-            _totalOperatorsHistory[quorumNumber][_totalOperatorsHistory[quorumNumber].length - 1].numOperators - 1 == operatorIdToSwapIndex, 
-            "IndexRegistry._processOperatorRemoval: operatorIdToSwap is not the last operator in the quorum"
-        );
-
+        uint32 currentNumOperators = _totalOperatorsHistory[quorumNumber][_totalOperatorsHistory[quorumNumber].length - 1].numOperators;
```

Comment:
neat! might want to make a getter/util like `numberOfOperators(uint8 quorumNumber)` ?

Reply:
good for clarity: https://github.com/Layr-Labs/eigenlayer-middleware/pull/31/commits/657c50c335526c4ca0cd0671c2bed90a1061ce2c

Reply:
> good for clarity: [657c50c](https://github.com/Layr-Labs/eigenlayer-middleware/commit/657c50c335526c4ca0cd0671c2bed90a1061ce2c)

wait, why not just make the existing 'view' method public?
I guess it's fine this way though 🤷

Reply:
i'm to sauce idk if we are cool with public functions?

Reply:
> i'm to sauce idk if we are cool with public functions?

IMO:
public 'view' functions are fine
public state-modifying functions are often potential landmines (it can be hard to tell when access control is correct on them, in particular)